src/pages/blog/proactive-data-security.pug

```diff
@@ -0,0 +1,107 @@
+
+          p.blog-post__text A seemingly small, but serious slip-up - like committing sensitive data to a GitHub repo, can have far reaching consequences. <a href="https://phys.org/news/2017-11-uber-breach.html" target="_blank">Uber learnt this the hard way.</a>
+
+          p.blog-post__text In 2017, Uber revealed that the personal information of 57 million users and 600,000 drivers were exposed. We later learnt that the data breach occurred due to sensitive data exposure. The hackers gained access to Uber’s GitHub account, where they found credentials that gave them access to AWS.
+
+          p.blog-post__text To make matters worse, Uber kept the breach under wraps for a year and only revealed it after paying the hackers $100,00 to destroy the data without any evidence proving that they did. After the breach and Uber’s handling thereof surfaced, Uber’s valuation dropped by 40%. 
+
+          p.blog-post__text Not prioritising security can be costly and the reputational damage could result in harsher financial damage than the actual breach. 
+
+          blockquote
+            q
+              | If your business were a house, proper, proactive security measures would be both your alarm system and your insurance.
+
+          p.blog-post__text Though out of the scope of this article, it’s important to highlight that these measures should include an effective disaster management strategy.
+
+          h3.blog-post__h3 Make use of modern frameworks
+
+          p.blog-post__text At Leaf we’re primarily using Ruby on Rails and React, both well established solutions with experienced core teams that are working hard to limit possible vulnerabilities and increase developer awareness. One strategy use by React is giving risky operations, that could potentially open you up for cyber attacks (like cross-site scripting or SQLinjection) names that give caution, like `dangerouslySetInnerHTML`.
+
+          p.blog-post__text Using modern frameworks is only one part of the equation. Keeping them up to date is the other. Outdated technologies generally come with increased operational costs and a dwindling talent pool, but they almost certainly come with a higher volume of vulnerabilities.
+
+          h3.blog-post__h3 Minimize the use of third-party packages
+
+          p.blog-post__text When it comes to dependencies, third-party packages and external scripts, be selective.
+
+          p.blog-post__text Towards the end of 2018 a stale, but popular <a href="https://github.com/dominictarr/event-stream/issues/116" target="_blank">npm package</a> with roughly 2 million downloads a week was <a href="https://snyk.io/blog/malicious-code-found-in-npm-package-event-stream/" target="_blank">hacked</a>. The hacker gained admin privileges to the repo by disguising himself as a willing maintainer and subsequently injected crypto-coin-stealing malware. The incident was a sobering reminder of just how fragile the ecosystem can be.
```

Comment:
Baby, baby recommendation, but I'd suggest either moving the opening comma to after '2018', or removing it altogether.